src/docs/truffle/how-to/migrate-to-hardhat.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Migrate to Hardhat
+layout: docs.hbs
+---
+
+# Migrate to Hardhat
+
+Like Truffle, Hardhat is a development environment for Ethereum software. It consists of different components for
+editing, compiling, debugging and deploying smart contracts and dapps.
+
+This topic provides some general steps you need to perform to migrate an existing Truffle project to Hardhat.
+
+## Install Hardhat
+
+Install Hardhat using the following command:
+
+```bash
+npm install --save-dev hardhat
+```
+
+## Update the folder structure
+
+Update the Truffle folder structure as follows:
+
+* Rename the `migrations` directory to `scripts`
+* Rename `truffle-config.js` to `hardhat.config.js`
```

Comment:
I think change line 26 to "Create a new `hardhat.config.js` with your network details and solidity settings. Refer to the Hardhat [docs](https://hardhat.org/hardhat-runner/docs/config) for advanced params".